apis/supernova/src/hooks.js

```diff
@@ -1143,14 +1143,26 @@ export function useEmitter() {
   return useInternalContext('emitter');
 }
 
+/**
+ * Blur function
+ * @callback BlurFunction
+ * @param {boolean} resetFocus parameter to determines whether the chart should be focused after blurring from inside
+ */
+
+/**
+ * Focus Selection function
+ * @callback FocusSelectionFunction
+ * @param {boolean} focusLast parameter to determines if the selection toolbar should be focuses the first or last element in the toolbar. if true, the last item is focused.
+ */
+
```

Comment:
I'd rather not have these named like this, bloats the API. But not sure what the alternative is, i'll need to look.

Reply:
Added suggestion in different comment.